server/src/modules/users/infra/repos/implementations/mikro-user-repo.ts

```diff
@@ -41,7 +41,7 @@ export class MikroUserRepo implements UserRepo {
   }
 
   async findAll(): Promise<Array<User>> {
-    const users = await DB.usersEntityRepo.findAll()
+    const users: Array<UserEntity> = await DB.usersEntityRepo.findAll()
```

Comment:
do we need this explicit type annotation? or can the compiler infer from the return type of `usersEntityRepo.findAll()`?

Reply:
Don't think we need it.